frontend/app.ts

```diff
@@ -19,7 +19,9 @@ export function renderBlock(containerId: string, props: BlockProps) {
     };
 
     const element = createElement(Block, blockProps);
-    render(element, document.getElementById(containerId));
+    setTimeout( function() {
```

Comment:
I think it would be better to move this to `view.ts` since this fix is specific to the front of the site (assuming the issue only happens there). `renderBlock()` is meant to be used in other contexts as well, like development environments.

Reply:
Would you mind also adding an inline comment, possibly with a link to relevant issue/PR?

Reply:
Done!